tests/migration/migration.go

```diff
@@ -3414,3 +3404,20 @@ func haveMigrationState(matcher gomegatypes.GomegaMatcher) gomegatypes.GomegaMat
 		}),
 	}))
 }
+
+func withKernelBoot() libvmi.Option {
+	return func(vmi *v1.VirtualMachineInstance) {
+		kernelBootFirmware := utils.GetVMIKernelBootWithRandName().Spec.Domain.Firmware
```

Comment:
Really not great that we're using the vms-generator code for this...
`libvmi.WithKernelBootContainer()` doesn't do what we need here?

Reply:
Yes, so bad..
Actually, `libvmi.WithKernelBootContainer()` was merged after this commit was created :)
Besides that, looks like it is not enough since currently this func also sets kernel path and initrd path (which libvmi does not):

```
spec.Domain.Firmware.KernelBoot = &v1.KernelBoot{
		KernelArgs: kernelArgs,
		Container: &v1.KernelBootContainer{
			Image:      image,
			KernelPath: kernelPath,
			InitrdPath: initrdPath,
		},
	}
```